src/osd/PG.cc

```diff
@@ -4148,8 +4151,11 @@ int PG::build_scrub_map_chunk(
     return ret;
   }
 
+  get_pgbackend()->be_scan_list(map, ls, deep, seed, preempted, handle);
```

Comment:
before objects_list_range, i think we still check *preempted because there is chance to change(in WAIT_PUSHES or WAIT_LAST_UPDATE it maybe lost lock)